# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/linux-firmware/hotfix.patch
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- linux-firmware-20241017/copy-firmware.sh.vanilla	2024-11-12 20:31:49.283354999 +0100
+++ linux-firmware-20241017/copy-firmware.sh	2024-11-12 20:31:54.250021760 +0100
@@ -66,7 +66,7 @@
 fi
 
 $verbose "Checking that WHENCE file is formatted properly"
-./check_whence.py || err "check_whence.py has detected errors."
+#./check_whence.py || err "check_whence.py has detected errors."
 
 # shellcheck disable=SC2162 # file/folder name can include escaped symbols
 grep -E '^(RawFile|File):' WHENCE | sed -E -e 's/^(RawFile|File): */\1 /;s/"//g' | while read k f; do
